src/v2/es-utils.js

```diff
@@ -54,6 +54,37 @@ class ESClass {
       });
   }
 
+  async healthy(timeout, status) {
+    const resp = await this.client.cluster.health({
+      level: 'cluster',
+      waitForStatus: status || 'green',
+      timeout: timeout || '90s'
+    });
+    if (resp.hasOwnProperty('body') && resp.body.status !== 'red') {
+      return (resp.body);
+    }
+    const info = await this.client.info();
+    return Object.assign(resp, info);
```

Comment:
This method is called `healthy` but returns some sort of information if `body.status` is red?

Reply:
I was trying to piggyback the `info` data into the `health` data and send it back as one object ... I changed it to a separate function that checks for elasticsearch cluster info.